### PR TITLE
Fix/env

### PR DESCRIPTION
**Pull request by gabe-microsoft**
- Fixed command for building environment using environment.yml file
- Re-generated new lock files (new env tested in WSL2)

### CODE REVIEW

File: notebooks/README.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ conda create --name aurorabp --file conda-win-64.lock
 Alternatively, if you're using a different platform, or have trouble installing using the conda-lock file, as described above, you can install the latest versions of all dependencies using the `environment.yml` file:
 
 ```bash
-conda create --name aurorabp --file environment.yml
+conda env create --name aurorabp --file environment.yml
```
Comment (jeremiahwander) [inline notebooks/README.md:32]: this `environment.yml` includes a [platform specification](https://conda-incubator.github.io/conda-lock/src_environment_yml/). When running this command that generates a warning message 

```text
EnvironmentSectionNotValid: The following section on '/home/miah/repos/aurorabp-sample-data/notebooks/environment.yml' is invalid and will be ignored:
 - platforms
```

A quick search didn't turn up any results as to how to suppress in conda, and the section seems necessary for lock file generation via conda-lock for non-host platforms. If you happen to know a magic way to convince conda to ignore that section in `environment.yml` let's implement it, else we can just live with the warning message.

One other alternative (that I don't really like) is to add an `environment-dev.yml` that includes the platforms section and remove the platforms section from the base `environment.yml`. This puts the onus on us to make sure those stay in sync, but does make the warning go away.

**Review by jeremiahwander — APPROVED** [review]
See comment for optional feedback.

File: notebooks/README.md (hunk 1 above)
Comment (gabe-microsoft) [reply notebooks/README.md:32]: Just pushed a change that both solves this issue and will make it easier for us to re-generate new lock files in the future. Basically, I removed the platform spec from environment.yml and added a script which we can run to update the lock files (the platforms are a constant in the script and specified on the command line).

Comment (jeremiahwander) [reply notebooks/README.md:32]: Great solution, resolving.

**Pull request merged**